jykuo-love-shiritori/twp
# [frontend] Admin ManageUser Section

Resolves #13 partially

### Changes


#### (J) Admin ManageUser
- [x] [ /admin/manageUser](http://localhost:5173/admin)
  - function: this is where admins commit murder🔪  at.
  - u should check:
    - the layout of both **big** and **small** screen.
    - **gear** and **trash can** button behalf.
    - **page** component and **confirm** button behalf.
    - the methods (using button in dropdown nav bar) of navigating to this page.

### Testing Approach (if unconventional)


- start the frontend server
  ```sh
  cd frontend
  npm run dev
  ```
- go to the new page
  - using website
    1. click the ``user icon`` dropdown list in ``nav bar`` at top left of ur screen.
    2. click the ``admin`` in the list.
    3. check everything 😳.
  - using [url](http://localhost:5173/admin)

### Additional context


- didnt implement functions of some components
  - page component, comfirm button
